apps/contracts/src/contracts/SlmStakerManager.sol

```diff
@@ -82,8 +82,8 @@ contract SlmStakerManager is Ownable {
         stakerStorage.pushUnstakedInfo(backer, beneficiary, userStake, block.timestamp);
         stakerStorage.sendFunds(backer, userStake);
 
-        for(uint256 i = 0; i < stakerPool.length; i += 1) {
-            if(stakerPool[i] == beneficiary) {
+        for (uint256 i = 0; i < stakerPool.length; i += 1) {
+            if (stakerPool[i] == beneficiary) {
                 delete stakerPool[i];
```

Comment:
Does this leave weird gaps in the array? I have some thoughts about how to solve this type of problem, and I think there are libraries, though the gas cost is non-trivial.

Reply:
Perhaps we can discuss this handling in more detail.